### PR TITLE
konvoy: changing default value to --with-aws-bootstrap-credentials=false

### PR DESCRIPTION
**Pull request by dkoshkin**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-89303

## Description of changes being made

Documenting the change of the default value of `--with-aws-bootstrap-credentials` to `false`. Code changes in https://github.com/mesosphere/konvoy2/pull/1279 

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4438.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

**Review by cneth — APPROVED** [review]

File: pages/dkp/konvoy/2.3/choose-infrastructure/aks/aks-advanced/aa-bootstrap/index.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Before you begin, you must:
 1.  Create a bootstrap cluster:
 
     ```bash
-    dkp create bootstrap --kubeconfig $HOME/.kube/config
+    dkp create bootstrap --with-aws-bootstrap-credentials=true --kubeconfig $HOME/.kube/config
```
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.3/choose-infrastructure/aks/aks-advanced/aa-bootstrap/index.md:26]: Not sure If I understood this the wrong way, but the command mentions AWS (to use AWS credentials), but in this file we are talking about AKS infrastructure? Should that not be AKS credentials?

File: pages/dkp/konvoy/2.3/choose-infrastructure/aks/aks-quickstart/index.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ Before starting the Konvoy installation, verify that you have:
 1.  Create a bootstrap cluster:
 
     ```bash
-    dkp create bootstrap --kubeconfig $HOME/.kube/config
+    dkp create bootstrap --with-aws-bootstrap-credentials=true --kubeconfig $HOME/.kube/config
```
Comment (gvasquezvargas) [inline pages/dkp/konvoy/2.3/choose-infrastructure/aks/aks-quickstart/index.md:93]: Same as above

File: pages/dkp/konvoy/2.3/release-notes/2.3.0/index.md [hunk 3]
```diff
@@ -37,6 +37,11 @@ The following features and capabilities are new for Version 2.2.
 
 ### Feature 1
 
+## Deprecations
+
+### Flag default changes
+
+The default value for flag `--with-aws-bootstrap-credentials` changed from `true` to `false`. Set the value to `true` to use AWS credentials from your environment, otherwise the AWS instance profile of the instance where the CAPA controller is running will be used.
```
[the remark this answers is not in the document]
Comment (mmcelroy91) [reply pages/dkp/konvoy/2.3/release-notes/2.3.0/index.md:44]: Not sure if the word instance used twice is necessary.  Please feel free to ignore if your words are more correct than mine.

**Review by mmcelroy91 — APPROVED** [review]

File: pages/dkp/konvoy/2.3/choose-infrastructure/aks/aks-advanced/aa-bootstrap/index.md (hunk 1 above)
Comment (dkoshkin) [reply pages/dkp/konvoy/2.3/choose-infrastructure/aks/aks-advanced/aa-bootstrap/index.md:26]: Good catch, I meant to add this to EKS 🙇

**Pull request merged**